src/hooks/useAssetLogoSource.ts

```diff
@@ -66,17 +65,13 @@ export default function useAssetLogoSource(
   isNative?: boolean,
   backupImg?: string | null
 ): [string | undefined, () => void] {
-  const showLogo = Boolean((address && checkWarning(address, chainId) === null) || isNative)
```

Comment:
I think we added this due to this PR:

https://linear.app/uniswap/issue/WEB-2119/dont-show-token-logos-for-tokens-on-the-unsupported-list

Matteen Mobasheran would have more info but basically some sort of IP concern with showing unsupported tokens, and it was meant to use the list in the checkWarning helper to filter.